platform/security/sts/security-sts-guestvalidator/src/main/java/org/codice/ddf/security/validator/guest/GuestValidator.java

```diff
@@ -65,10 +57,11 @@ private GuestAuthenticationToken getGuestTokenFromTarget(ReceivedToken validateT
     }
 
     private boolean validIpAddress(String address) {
-        return IPV4_PATTERN.matcher(address)
-                .matches() || IPV6_STD_PATTERN.matcher(address)
-                .matches() || IPV6_HEX_COMPRESSED_PATTERN.matcher(address)
-                .matches();
+        try {
+            return InetAddress.getByName(address) != null;
+        } catch (UnknownHostException e) {
```

Comment:
Any reasonable chance that a security exception could be thrown? "SecurityException - if a security manager exists and its checkConnect method doesn't allow the operation"